src/components/pickers/BasePicker.tsx

```diff
@@ -353,4 +355,48 @@ export class BasePicker<T, P extends IBasePickerProps<T>> extends BaseComponent<
       this._removeItems(this._selection.getSelection());
     }
   }
-}
\ No newline at end of file
+}
+
+export class BasePickerListBelow<T, P extends IBasePickerProps<T>> extends BasePicker<T, P> {
+  public render() {
+    let { displayValue } = this.state;
+
+    return (
+      <div>
+        <div ref={ this._resolveRef('_root') }
+          className={ css('ms-BasePicker', this.props.className ? this.props.className : '') }
+          onKeyDown={ this._onKeyDown }>
+          <SelectionZone selection={ this._selection }
+            selectionMode={SelectionMode.multiple}>
+            <div className='ms-BasePicker-text'>
+              <input ref={ this._resolveRef('_input') }
+                onFocus={ this._onInputFocus }
+                onChange={ this._onInputChange }
+                value={ displayValue }
+                className='ms-BasePicker-input'
+                />
+            </div>
+          </SelectionZone>
+        </div>
+        { this._renderSuggestions() }
+        <FocusZone ref={ this._resolveRef('_focusZone') } className='ms-BasePicker-selectedItems'>
+          { this._renderItems() }
+        </FocusZone>
+
+      </div>
+    );
+  }
+
+  protected _onBackspace(ev: React.KeyboardEvent) {
```

Comment:
it is unclear why this is protected.

1. can you add comments what this is and why its protected

2. can you remove the underscore (I know this is breaking from consistency, but i think we need to consider protected things as part of the public contract.)